### PR TITLE
tests: enable --analyze tests for windows

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
tests

**Did you add tests for your changes?**
yes
**If relevant, did you update the documentation?**
No
**Summary**
fix for windows.

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

**Review by evilebottnawi — DISMISSED** [review]

File: test/analyze/webpack.config.js [hunk 1]
```diff
@@ -2,6 +2,6 @@ const WebpackCLITestPlugin = require('../utils/webpack-cli-test-plugin');
 
 module.exports = {
     mode: 'development',
-    entry: './main.js',
+    entry: './src/main.js',
```
Comment (rishabh3112) [inline test/analyze/webpack.config.js:5]:
```suggestion
```

File: test/analyze/src/main.js [hunk 2]
```diff
@@ -0,0 +1 @@
+console.log('analyze flag test');
```
Comment (rishabh3112) [inline test/analyze/src/main.js:1]: Make `src/index.js` file instead.

**Review by rishabh3112 — CHANGES_REQUESTED** [review]
Nice Job! 
Left a small suggestion.`main.js` is default output of `webpack`, should keep file names less ambiguous in tests.

File: test/analyze/webpack.config.js (hunk 1 above)
Comment (snitin315) [reply test/analyze/webpack.config.js]: done

File: test/analyze/src/main.js (hunk 2 above)
Comment (snitin315) [reply test/analyze/src/main.js]: Done

**Review by rishabh3112 — APPROVED** [review]

**Review by evilebottnawi — APPROVED** [review]

**Pull request merged**